### DISCUSSION TITLE
Emojis break block content

### DISCUSSION THREAD

**Discussion by byjacobbyers** [sanity-io/sanity#3455] [Sanity Studio Support]
Hey! Whenever I add emojis to a portable text block in Sanity, I get the error "URI Malformed" when I try to edit text. It goes away when I remove the emoji and works as intended there after. Not sure if there is a way to fix this.

**Comment by rexxars** [comment]
Thanks for reporting - this is an issue with the [diff-match-patch](https://github.com/google/diff-match-patch) library that we are using. We are [working on a fork](https://github.com/sanity-io/diff-match-patch/tree/fix/surrogate-pairs-2), but there are still some issues to be ironed out. Please bear with us! ❤️
